compute/re-create-instance-from-volume.rst

```diff
@@ -0,0 +1,45 @@
+=============================
+Recreate instance from volume
+=============================
+
+Sometimes its necessary to re-create an Openstack instance from a current volume. The process looks like this:
```

Comment:
inconsistent `re-create` vs `recreate` in title and document

Reply:
starting sentence with ``You can recreate...`` is perhaps more clean than starting it with ``Sometimes...``